### ISSUE TITLE
Copying resources into the destination directory

### ISSUE THREAD

**Issue by dvberkel** [azerupi/mdBook#441]
**TL;DR.**  Is copying resources into the destination directory a feature @azerupi would want?

At the moment images need to have a standalone URL in order for them to be shown. When the files are served, e.g. from `mdbook serve` or via GitHub pages, they could be fetched from the same server. 
 
I would therefore like to be able to configure mdbook to copy a resource directory into the destination directory. This would allow resources to be served, simplifying the publishing process.

I looked at the API, specifically at the various configurations, and it seems that this feature is not supported. Is this something that is interesting to support?

**Comment by azerupi** [comment]
All files placed in your source directory should be automatically copied over to the output directory. So if you place your images in `src/img/...` everything should work.

Is that what you wanted or did I misunderstand? :)

**Comment by dvberkel** [comment]
This is exactly what I wanted! :1st_place_medal:
